### PR TITLE
fix(mobile): Generate 1 splash screen on Android

### LINKED ISSUE #982: [QUESTION] There are two splash screens on Android 12 and higher
**Describe the bug**
I've recently accidentally updated my Android version to Android v12.
This version apparently had a new 'splash screen' feature, which forced a splash screen to every app- regardless of whether the app supports it or not. More info: https://developer.android.com/develop/ui/views/launch/splash-screen 

I've noticed that when I open Immich, I have two splash screens. One of which was forced by Android 12, and the other built-in splash screen. Android does have an API to integrate the app's splash screen with the splash screen forced by Android 12, so this is fixable. I've also added a video recording.

**Task List**

*Please complete the task list below. We need this information to help us reproduce the bug or point out problems in your setup. You are not providing enough info may delay our effort to help you.*

- [x] I have read thoroughly the README setup and installation instructions.
- [ ] I have included my `docker-compose` file.
- [ ] I have included my redacted `.env` file.
- [x] I have included information on my machine, and environment.

**To Reproduce**
1. Update to Android 12
2. Open Immich
3. See two splash screens

**Expected behavior**
I expected Immich to have one splash screen instead of two.

**Screenshots**

https://user-images.githubusercontent.com/47184046/202293016-a41743a1-20c8-454c-adb7-25cf501157b2.mp4

**System**
- Phone OS [iOS, Android]: `Android 12`
- Server Version: `1.35.0`
- Mobile App Version: `1.35.0`

### PR DESCRIPTION
**Pull request by martyfuhry**
- Adds `flutter_native_splash` dependency and build script 
- Required adding `image:` to the `dependency_overrides` since `flutter_launcher_icons` is out of date
- Adds `create_splash` command to `makefile` to run the `flutter_native_splash` create script
- Adds `assets/immich-logo-no-outline-android12.png` which is 1152x1152 pixels with the app icon centered within a 640 pixel diameter circle to match the [Android 12 splash screens](https://developer.android.com/develop/ui/views/launch/splash-screen) requirements
- And changes initial splash screen to no longer contain the "Immich" text.

The initial splash screen no longer contains the text of the app name since that would require adding the text to the splash screen image. If that's desired, I can work on that.

Closes #982

### CODE REVIEW

File: mobile/ios/Runner/Info.plist [hunk 1]
```diff
@@ -1,105 +1,97 @@
...
+			<string>ja</string>
+			<string>ko</string>
+			<string>nl</string>
+			<string>pl</string>
+			<string>pt</string>
+			<string>ru</string>
+			<string>sk</string>
+			<string>zh</string>
+		</array>
+		<key>UIStatusBarHidden</key>
```
Comment (alextran1502) [inline mobile/ios/Runner/Info.plist:94]: Is this auto generated by some process in this PR?

**Comment by alextran1502** [comment]
Can you help me fix the `Info.plist` conflict?

File: mobile/ios/Runner/Info.plist (hunk 1 above)
Comment (martyfuhry) [reply mobile/ios/Runner/Info.plist:94]: Odd, it might be. Let me take a look.

Comment (martyfuhry) [reply mobile/ios/Runner/Info.plist:94]: Yup, I just reran the script `flutter pub run flutter_native_splash:create` (from the `makefile`) and this line shows up in the output:

`[iOS] Updating ios/Runner/Info.plist for status bar hidden/visible`

I suppose that answers our question. It sure generates a terrible diff. I think it's just an indentation, but I didn't read it very carefully. The main relevant line it seems to be adding is here at the end:

```
<key>UIStatusBarHidden</key>
<false/>
```

**Comment by martyfuhry** [comment]
I rebased on `main`, that should do it. The only real (non-generated) changes to this PR are:

1. Added `flutter_native_splash` as a `dev_dependency`
2. Added `create_splash` to the `makefile`
3. Changed the `SplashScreenPage` to no longer show the "Immich" text (which may be a bigger change than you like)

Everything else was generated by `make create_splash`.

If you'd like to remain using the "Immich" text, you'll need to add that to the initial splash screen image to avoid the "double splash screen" effect from the inherent issue. You want your splash asset to exactly match the first frame that Flutter is going to draw in your app, which happens to be the `SplashScreenPage` in order to appear seamless. 

Someone should also test iOS to make sure the splash screens / assets look OK to them.

**Comment by martyfuhry** [comment]
https://user-images.githubusercontent.com/100457/215288504-f7b6c88b-9c6f-4894-8a97-01f47cc099fb.mp4

Here's how it looks on my Android 12 device. Notice how the splash screen exactly matches the first frame from Immich. You could also animate the Splash to Login transition with a Hero animation pretty easily.

**Pull request merged**